app/helpers/application_helper.rb

```diff
@@ -23,7 +23,11 @@ def avatar(email = nil)
     else
       gravatar_id = Digest::MD5.hexdigest(email.downcase)
       gravatar_size = Rails.application.config.gravatar_size
-      tag :image, src: "http://gravatar.com/avatar/#{gravatar_id}.png?s=#{gravatar_size}&d=#{CGI.escape(root_url + default)}"
+      if Rails.env.development?
+        tag :image, src: "http://gravatar.com/avatar/#{gravatar_id}.png?s=#{gravatar_size}&d=#{CGI.escape(default)}"
+      else
+        tag :image, src: "http://gravatar.com/avatar/#{gravatar_id}.png?s=#{gravatar_size}&d=#{CGI.escape(root_url + default)}"
+      end  
```

Comment:
This'll work, but just some constructive criticism: this isn't very DRY - I'd suggest using the environment check to set a variable (`gravatar_default` or something similar) and use that variable in the image src. That'll cause less repetition, and make the URL just a bit easier to read.